content/en/docs/installation/armory-operator/op-manifest-reference/armory.md

```diff
@@ -2,11 +2,13 @@
 title: Armory Config
 weight: 2
 description: >
-  This page describes `spec.spinnakerConfig.config.armory`.
+  This page describes the `spec.spinnakerConfig.config.armory` section, which is used to configure proprietary features in Armory Enterprise for Spinnaker.
```

Comment:
```suggestion
  This page describes the `spec.spinnakerConfig.config.armory` section, which is used to configure features in Armory Enterprise for Spinnaker.
```

Suggested because the proprietary callout immediately following could sound redundant with both instances.